specification/udp/src/main/scripts/org/kaazing/specification/udp/rfc768/concurrent.writes.together/server.rpt

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+accept udp://localhost:8080
+       notify BOUND
+
+# Stream 1
+accepted
+connected
+read /(?<hello1>[a-zA-Z0-9]+)/
```

Comment:
I would recommend having the clients write text like "1Hello", "1Bye", with the variable part at the start, then you can make the RE expressions tighter, like
```
read /(?<hello1>[123]Hello)/
```
That will eliminate any possible issues like packets being coalesced, or k3po getting confused because of fragmentation, and also makes the script easy to understand.